portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,51 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>My Portfolio</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+	<div id="content">
+		<div id="white-box">
+			<h1>Caroline's Portfolio</h1>
+			<div id="blurb">
+				<p>I'm Caroline and this is my portfolio! A bit about me: I'm currtenly a rising Junior at Cornell's
+					College
+					of
```

Comment:
this section is hard to follow in code due to code formatting. This YAQS suggests https://yaqs.corp.google.com/eng/q/5051660848332800 a line limit of 100, could you try to keep it to that?

Reply:
format has been fixed!